test/unit/core/type_traits/template_inspection_test.cpp

```diff
@@ -9,11 +9,22 @@
 
 #include <gtest/gtest.h>
 
-#include <seqan3/core/metafunction/template_inspection.hpp>
+#include <seqan3/core/type_traits/concept.hpp>
+#include <seqan3/core/type_traits/template_inspection.hpp>
 #include <seqan3/core/type_list.hpp>
 
 using namespace seqan3;
 
+TEST(template_inspect, concept)
```

Comment:
we name the test `concept_check` through out the library

Reply:
well not everywhere but in most places. I changed it accordingly